cypress/fixtures/IntegrationMultiFlow.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: camel.apache.org/v1
+kind: Integration
+metadata:
+  name: ''
+spec:
+  flows:
+  - from:
+      uri: cron:cron
+      parameters:
+        schedule: '1000'
+      steps:
+      - set-body:
+          simple: body
+      - to:
+          uri: log:log1
+---
+apiVersion: camel.apache.org/v1
+kind: Integration
+metadata:
+  name: ''
+spec:
+  flows:
+  - from:
+      uri: timer:test
+      steps:
+      - to:
+          uri: log:log2
```

Comment:
This integration type will be slightly different from what it is now. Later on, multiple routes from the `Integration` type will be handled inside of the `flows` array but this is in the works at the backend.

Can I suggest you another fixture in the meantime? This is a multiple Camel route fixture.

```yaml
- from:
    uri: cron:cron
    parameters:
      schedule: '1000'
    steps:
    - set-body:
        simple: body
    - to:
        uri: log:log1

- from:
    uri: timer:test
    steps:
    - to:
        uri: log:log2
```

Reply:
sure, thanks for letting me know - will convert this to the `flows` once it's in the backend. Also the second fixture - I'll extend the tests and add there also camelRoute type tests, so the covered area represents all the possible route options.